Tasks/MavenV2/task.json

```
@@ -227,6 +227,7 @@
       "visibleRule": "javaHomeSelection = JDKVersion",
       "options": {
         "default": "default",
+        "1.17": "JDK 21",
```

```suggestion
        "1.21": "JDK 21",
```
Copy and paste error? 😄

---

Yup. Sorry and thank you @TotallyMehis